SeaPublicWebsite.UnitTests/QuestionFlowServiceTests.cs

```diff
@@ -1943,63 +1943,6 @@ public void RunSkipLinkTestCases(QuestionFlowServiceTestCase testCase)
                 nameof(EnergyEfficiencyController.PropertyType_Get),
                 "EnergyEfficiency",
                 new { reference = "ABCDEFGH" }
-            )),
-        new(
-            "Home age skips to wall construction",
-            new Input(
-                QuestionFlowPage.HomeAge,
-                "ABCDEFGH"
-            ),
-            new PathByActionArguments(
-                nameof(EnergyEfficiencyController.WallConstruction_Get),
-                "EnergyEfficiency",
-                new { reference = "ABCDEFGH" }
-            )),
```

Comment:
All these tests are no longer applicable since we treat skip links as post actions which set answers to null.